### ISSUE TITLE
Fairy Ring Plugin: DKP code incorrect description

### ISSUE THREAD

**Issue by Hacksaw140** [runelite/runelite#14656]
Please check if your issue is not a duplicate by [searching existing issues](https://github.com/runelite/runelite/search?type=Issues)

**Describe the bug**
The Fairy ring plugin is displaying an outdated description for the DKP location. Some time ago it was changed to "Karamja: Gnome Glider" however the plugin still displays "South of Musa Point"

**To Reproduce**
1. Open the Fairy ring interface with the plugin active
2. Input DKP into the fairy ring

**Expected behavior**
Fairy Ring plugin should display "Gnome Glider"

**Screenshots**
N/A

**Environment (please complete the following information):**
 - OS: Windows
 - RuneLite version: most recent
 - Launcher version: most recent

**Additional context**
N/a

**Comment by JakeDeakin** [comment]
Quick edit to the FairyRings file in the plugin was all that was needed. Spotted the Farming Guild/Mount Karuulm ring CIR was also wrong so changed that as well.

Pull Request here #14657

**Comment by Adam-** [comment]
Many of our fairy ring descriptions are not exact matches of the game descriptions, I don't think they have to be exact either. In this case "south of musa point" seems much more useful than "gnome glider" since there are 11 (7 active and 4 crashed).. This one in particular is crashed which makes it even less useful.

**Comment by JakeDeakin** [comment]
That's true. Though these were the only 2 I saw on the list that weren't the same as the in game description. If it's an unnecessary change though happy to just delete it.
